docs/src/main/asciidoc/ribbon-integration.adoc

```diff
@@ -1,40 +1,54 @@
 == Ribbon discovery in Kubernetes
 
 
-Spring Cloud client applications calling a microservice should be interested on relying on a client load-balancing
+Spring Cloud client applications that call a microservice should be interested on relying on a client load-balancing
 feature in order to automatically discover at which endpoint(s) it can reach a given service. This mechanism has been
-implemented within the [spring-cloud-kubernetes-ribbon](spring-cloud-kubernetes-ribbon/pom.xml) project where a
-Kubernetes client will populate a https://github.com/Netflix/ribbon[Ribbon] `ServerList` containing information
+implemented within the https://github.com/fabric8io/spring-cloud-kubernetes/tree/master/spring-cloud-kubernetes-ribbon[spring-cloud-kubernetes-ribbon] project, where a
```

Comment:
In the URL, the path segment `fabric8io` should be `spring-cloud`.

Reply:
Fixed. Thanks.